solarforecastarbiter/tests/test_cli.py

```diff
@@ -216,7 +216,7 @@ def test_reference_nwp(cli_token, mocker):
                             ['-u user', '-p pass', '--run-time=20190501T1200Z',
                              '--issue-time-buffer=2h',
                              tmpdir])
-        assert cli.nwp.BASE_PATH == tmpdir
+        assert cli.nwp.BASE_PATH.replace('/private', '') == tmpdir
```

Comment:
Why is the replacement necessary? Does ``tempfile.TemporaryDirectory()`` not always return the correct directory name?

Reply:
This seems to be a requirement on my mac. @dplarson Have you run into this on yours?

Reply:
Ah, I see https://stackoverflow.com/questions/12482702/pythons-os-chdir-and-os-getcwd-mismatch-when-using-tempfile-mkdtemp-on-ma
Then I suggest 
```suggestion
        assert cli.nwp.BASE_PATH == Path(tmpdir).resolve()
```
after import Path from pathlib

Reply:
I just tried on my machine (macOS 10.14.6, conda env with python 3.7.4) and I also get an error on this line unless I add the `.repalce("/private", "")`. Here's the error I see when I run `pytest -v -s -x test_cli.py`:

```shell
>           assert cli.nwp.BASE_PATH == tmpdir
E           AssertionError: assert '/private/var...T/tmp83um2br6' == '/var/folders/...T/tmp83um2br6'
E             - /private/var/folders/4j/07gz5y8x6_g2szmj972sgzzr0000gp/T/tmp83um2br6
E             ? --------
E             + /var/folders/4j/07gz5y8x6_g2szmj972sgzzr0000gp/T/tmp83um2br6

test_cli.py:219: AssertionError
```

Reply:
maybe need to turn it back into a string too

Reply:
I've been seeing this issue for a few months on my mac. I'm confused about it because it worked when I first wrote the test, but happy to see the fix and move on.